libphal/phal_sbe.C

```diff
@@ -288,6 +288,16 @@ void getDump(struct pdbg_target *proc, const uint8_t type, const uint8_t clock,
 	log(level::INFO, "Enter: getDump(%d) on %s", type,
 	    pdbg_target_path(proc));
 
+	// Validate input target is processor target.
+	validateProcTgt(proc);
+
+	if (!isTgtPresent(proc)) {
+		log(level::ERROR, "getDump(%s) Target is not present",
+		    pdbg_target_path(proc));
+	}
+	// SBE halt state need recovery before dump chip-ops
+	sbeHaltStateRecovery(proc);
```

Comment:
this one will set the state to RUNTIME and below check check in validateSBEState checks if it is BOOTED

Reply:
sbe message  regiuster current state gets updated to Runnig and Booted bit is diffent. Validate looks msb bit ( sbe booted bit).

Reply:
This brought up good poiint , i should updatde sbe state to check cfam, if the halt state is not back,